README.md

```diff
@@ -388,8 +388,8 @@ That's all folks! :+1:
 - [React Native](./packages/rudy/docs/react-native.md)
 - [React Navigation Support](./packages/rudy/docs/react-native.md#first-class-react-navigation-support)
   🔮 - _my current primary day-to-day focus_
-- [Prefetching!](./packages/rudy/docs/prefetching.md) 🔮 - `<Link prefetch />` **_powered
```

Comment:
still reading through the code but we should make a do too, or help wanted on features, preloading we can now use webpack magic comments if the babel-universal plugin is fully up to date. 

i think at the time, those magic comments didn't exist 

https://github.com/faceyspacey/babel-plugin-universal-import/pull/69

IF we dont have a, help needed on. ```.md``` file, then we can backlog this.

Reply:
Sure - is there anything extra to do here to support it or can you simply use the magic comments where you call `import()`?

This change is just prettier, I didn't touch this myself.

Reply:
As long as it’s webpack:). All good